notebooks/notebooks/iclr2022/estimate-flops-katago.ipynb

```diff
@@ -0,0 +1,662 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    visits_per_row = 1000
</pre><p>Do we know how the # of visits varied throughout training? I seem to recall we asked lightvector but I can't find this now so perhaps am misremembering. There is https://discord.com/channels/417022162348802048/583775968804732928/860157921710178345 which indicate it's not fixed at any given point in time and additional may vary over time:</p><blockquote> (For the 6-block networks at the time, Kata was using 600 max visits, 100 cheap)</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Ah we don't know. I just took the 1000 from the default config here: https://github.com/lightvector/KataGo/blob/master/cpp/configs/training/selfplay8b20.cfg

But e.g. https://github.com/lightvector/KataGo/blob/master/cpp/configs/training/selfplay8a.cfg has maxVisits=600 instead.

I will ask lightvector again what he thinks a reasonable estimate is, and also link him to our notebook to see if our estimates seem kosher to him.

Reply:
<p>Asking lightvector seems good.</p>